MSAL/test/app/MSALTestAppSettings.m

```diff
@@ -314,4 +314,16 @@ + (BOOL)isSSOSeeding
     return [currentProfile isEqualToString:@"CompanyPortal"];
 }
 
++ (NSArray<NSString *> *)getSSOSeedingScope
+{
+    NSDictionary *currentProfile = [MSALTestAppSettings currentProfile];
+    NSMutableArray<NSString *> *ssoSeedingScopes = [NSMutableArray new];
+    [ssoSeedingScopes addObject:[currentProfile objectForKey:@"resourceId"]];
+    if ([ssoSeedingScopes count])
+    {
+        [ssoSeedingScopes addObject:@"01cb2876-7ebd-4aa4-9cc9-d28bd4d359a9/.default"];
```

Comment:
nit: add a comment that this is DRS resource. Will be a huge favor for developers looking at it :)

Reply:
Address in next commit.